src/views/templates/view_steps_agreement.jade

```diff
@@ -1,5 +1,6 @@
 extends _base
 
+
```

Comment:
We may avoid commits for just adding blank lines. Instead we can amend our previous commit :
```bash
# Add your modifications
git add -p
git commit --amend
```